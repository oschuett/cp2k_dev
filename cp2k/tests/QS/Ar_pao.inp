&CP2K
 PROGRAM      Quickstep
 IOLEVEL      10
 FFTLIB       FFTSG
&END

&DFT
 FUNCTIONAL   Pade
&END

&PAO
 SUBSPACE_POS_PRECISION  1.0e-6
 DO_TESTS
&END

&QS
 CUTOFF       300
 EPS_DEFAULT  1.0E-12
 EPS_RHO      1.0E-8
 EPS_GVG      1.0E-6
 REL_CUTOFF   30
 PAO
&END

&SCF
 GUESS        atomic
 EPS_DIIS     0.1
 MAX_DIIS     4
 EPS_SCF      1.0E-6
 MAX_SCF      30
 MIXING       0.4
&END

&PRINT medium
&END

&KIND Ar
 BASIS_SET    DZVP-GTH-PADE
 POTENTIAL    GTH-PADE-q8
 PAO_MIN_BAS  4 3 2 1
&END

&CELL
 UNIT         ANGSTROM
 ABC          12.0  12.0  12.0
&END

&COORD
 Ar     0.000000  0.000000  0.000000
&END
