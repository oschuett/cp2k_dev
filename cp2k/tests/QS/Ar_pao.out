
  **** **** ******  **  PROGRAM STARTED AT              Fri Jul  5 16:32:56 2002
 ***** ** ***  *** **   PROGRAM STARTED ON                       muralto.cscs.ch
 **    ****   ******    PROGRAM STARTED BY                                 fawzi
 ***** **    ** ** **   PROGRAM PROCESS ID                                 10055
  **** **  *******  **  PROGRAM STARTED IN             /home/fawzi/cp2k/tests/QS

 CP2K| Program compiled at                         Wed Jun 26 17:12:21 CEST 2002
 CP2K| Program compiled on                                       muralto.cscs.ch
 CP2K| Program compiled for                                       Linux-i686-pgi
 CP2K| Program name                                                           QS
 CP2K| Input file name                                                Ar_pao.inp
 CP2K| Project name                                                      project
 CP2K| Default FFT library                                                 FFTSG
 CP2K| Default basis set file                                          BASIS_SET
 CP2K| Default external potential file                                 POTENTIAL
 CP2K| Default library path                                                   ./
 CP2K| Global print level                                                     10
 CP2K| Total number of processors                                              1
 CP2K| This output from processor                                              0

 *******************************************************************************
 *******************************************************************************
 **                                                                           **
 **     #####                         ##              ##                      **
 **    ##   ##            ##          ##              ##                      **
 **   ##     ##                       ##            ######                    **
 **   ##     ##  ##   ##  ##   #####  ##  ##   ####   ##    #####    #####    **
 **   ##     ##  ##   ##  ##  ##      ## ##   ##      ##   ##   ##  ##   ##   **
 **   ##  ## ##  ##   ##  ##  ##      ####     ###    ##   ######   ######    **
 **    ##  ###   ##   ##  ##  ##      ## ##      ##   ##   ##       ##        **
 **     #######   #####   ##   #####  ##  ##  ####    ##    #####   ##        **
 **           ##                                                    ##        **
 **                                                                           **
 **                                                ... make the atoms dance   **
 **                                                                           **
 **                         Version 3.1 (June 2002)                           **
 **                                                                           **
 **            Copyright (C) by CP2K Developers Group (2000 - 2002)           **
 **                                                                           **
 *******************************************************************************
 *******************************************************************************

 DFT| Force calculation                                                       NO
 DFT| Cutoffs: density                                               0.10000E-09
 DFT|          gradient                                              0.10000E-07
 DFT| XC mesh (PW density)                                               default
 FUNCTIONAL| Exchange Functional                                            NONE
                                                                            NONE
 FUNCTIONAL| Correlation Functional                                         PADE
              S. Goedecker, M. Teter and J. Hutter, Phys. Rev. B 54, 1703 (1996)

 QS| Method:                                                                 GPW
 QS| Number of grid levels:                                                    4
 QS| Density cutoff [a.u.]: 1) grid level                                  150.0
 QS|                        2) grid level                                   75.0
 QS|                        3) grid level                                   37.5
 QS|                        4) grid level                                   18.8
 QS| PAO active
 QS| Grid level progression factor:                                          2.0
 QS| Relative density cutoff [a.u.]:                                        30.0
 QS| Interaction thresholds: eps_pgf_orb:                                1.0E-12
 QS|                         eps_core_charge:                            1.0E-12
 QS|                         eps_rho_gspace:                             1.0E-08
 QS|                         eps_rho_rspace:                             1.0E-08
 QS|                         eps_gvg_rspace:                             1.0E-06
 QS|                         eps_ppl:                                    1.0E-12
 QS|                         eps_ppnl:                                   1.0E-12

 CELL| Volume [ANGSTROM**3]:                                            1728.000
 CELL| Vector a [ANGSTROM]:      12.000     0.000     0.000    |a| =      12.000
 CELL| Vector b [ANGSTROM]:       0.000    12.000     0.000    |b| =      12.000
 CELL| Vector c [ANGSTROM]:       0.000     0.000    12.000    |c| =      12.000
 CELL| Angle (b,c) [degree]:                                              90.000
 CELL| Angle (a,c) [degree]:                                              90.000
 CELL| Angle (a,b) [degree]:                                              90.000
 CELL| Grid size for subcell generation                                        2


 BASIS SET INFORMATION (Unnormalised Gaussian-type functions)

  1. Atomic kind: Ar

   - GTO basis set information

     GTO basis set name: DZVP-GTH-PADE

     Set   Shell     n   l            Exponent    Coefficient

       1       1     3   0            2.643629       0.373762
                                      1.694896      -0.099656
                                      0.523111      -0.831520
                                      0.166331      -0.333993
       1       2     4   0            2.643629       0.000000
                                      1.694896       0.000000
                                      0.523111       0.000000
                                      0.166331       1.000000
       1       3     4   1            2.643629       0.173520
                                      1.694896      -0.310093
                                      0.523111      -0.580756
                                      0.166331      -0.387933
       1       4     5   1            2.643629       0.000000
                                      1.694896       0.000000
                                      0.523111       0.000000
                                      0.166331       1.000000

       2       1     3   2            0.850000       1.000000


 ATOMIC KIND INFORMATION

  1. Atomic kind: Ar

     Number of atoms of this atomic kind in the cell:      1

   - Orbital basis set information

     Orbital basis set name: DZVP-GTH-PADE

     Number of orbital shell sets:                 2
     Number of orbital shells:                     5
     Number of primitive Cartesian functions:      5
     Number of Cartesian basis functions:         14
     Number of spherical basis functions:         13

     Normalised Cartesian orbitals

     Set   Shell   Orbital            Exponent    Coefficient

       1       1    3s                2.643629       0.552275
                                      1.694896      -0.105505
                                      0.523111      -0.364525
                                      0.166331      -0.061998

       1       2    4s                2.643629       0.000000
                                      1.694896       0.000000
                                      0.523111       0.000000
                                      0.166331       0.185626

       1       3    4px               2.643629       0.833759
                                      1.694896      -0.854793
                                      0.523111      -0.368278
                                      0.166331      -0.058737
       1       3    4py               2.643629       0.833759
                                      1.694896      -0.854793
                                      0.523111      -0.368278
                                      0.166331      -0.058737
       1       3    4pz               2.643629       0.833759
                                      1.694896      -0.854793
                                      0.523111      -0.368278
                                      0.166331      -0.058737

       1       4    5px               2.643629       0.000000
                                      1.694896       0.000000
                                      0.523111       0.000000
                                      0.166331       0.151410
       1       4    5py               2.643629       0.000000
                                      1.694896       0.000000
                                      0.523111       0.000000
                                      0.166331       0.151410
       1       4    5pz               2.643629       0.000000
                                      1.694896       0.000000
                                      0.523111       0.000000
                                      0.166331       0.151410

       2       1    3dx2              0.850000       1.238490
       2       1    3dxy              0.850000       2.145128
       2       1    3dxz              0.850000       2.145128
       2       1    3dy2              0.850000       1.238490
       2       1    3dyz              0.850000       2.145128
       2       1    3dz2              0.850000       1.238490
 # projection indexes
     1
     2
     3
     4
 # non projection indexes
     5
     6
     7
     8
     9
    10
    11
    12
    13


 TOTAL NUMBERS AND MAXIMUM NUMBERS

  Total number of

  - Atomic kinds:                       1
  - Atoms:                              1
  - Shell sets:                         2
  - Shells:                             5
  - Primitive Cartesian functions:      5
  - Cartesian basis functions:         14
  - Spherical basis functions:         13

  Maximum angular momentum quantum number of the

  - Orbital basis functions:                        2
  - Local part of the GTH pseudopotential:          0
  - Non-local part of the GTH pseudopotential:      2


 ATOMIC COORDINATES IN ANGSTROM

  Atom  Kind  Element       X           Y           Z       Z(eff)    Mass

     1    1   Ar   18    0.000000    0.000000    0.000000      8    39.9480


 RADII OF THE ORBITAL BASIS IN ANGSTROM

  Kind   Label       Radius

     1   Ar         6.903966


 RADII OF THE CORE CHARGE DISTRIBUTIONS IN ANGSTROM

  Kind   Label       Radius

     1   Ar         1.631446


 RADII OF THE LOCAL PART OF THE GTH PSEUDOPOTENTIAL IN ANGSTROM

  Kind   Label       Radius

     1   Ar         1.631446


 SCF PARAMETERS

  Density guess: ATOMIC

  nrebuild:              1
  max_scf:              30
  max_diis:              4
  eps_scf:            1.00E-06
  eps_diis:           1.00E-01
  eps_eigval:         1.00E-05
  p_mix:              0.40
  work_syevx:         1.00
  level_shift [a.u.]: 0.00
  smear [a.u.]:       0.00
  WARNING cp_log_handling:cp_logger_init overwriting file pao_tests.log
 NOTE    in pao_read_section:pao_env_init_from_file,  performing pao tests
 NOTE    in
 pao_read_section:pao_env_init_from_file,  +++++
 all the pao tests were SUCCESSFULL ++++


 SUBCELL GRID FOR THE SAB_ORB NEIGHBOR LISTS

  Atomic kind pair          Grid size                Subcell size in ANGSTROM
         1       1        1    1    1         12.000000   12.000000   12.000000


 SUBCELL GRID FOR THE SAC_PPL NEIGHBOR LISTS

  Atomic kind pair          Grid size                Subcell size in ANGSTROM
         1       1        1    1    1         12.000000   12.000000   12.000000


 SUBCELL GRID FOR THE SCA_PPNL NEIGHBOR LISTS

  Atomic kind pair          Grid size                Subcell size in ANGSTROM
         1       1        1    1    1         12.000000   12.000000   12.000000


 DISTRIBUTION OF THE OVERLAP MATRIX ELEMENTS

  Image atoms included:

    PE   Matrix blocks   Matrix elements

     0               7              1183

   Sum               7              1183


  Image atoms not included:

    PE   Matrix blocks   Matrix elements

     0               1               169

   Sum               1               169

    of               1 blocks in the full matrix (100.0 % occupation)


 SCF WAVEFUNCTION OPTIMIZATION


 SUBCELL GRID FOR THE CALCULATION OF THE CORE CHARGE OVERLAP

  Atomic kind pair          Grid size                Subcell size in ANGSTROM
         1       1        4    4    4          3.000000    3.000000    3.000000

  Overlap energy of the core charge distribution:                  0.0000000000
  Self energy of the core charge distribution:                   -45.1351666838

  Number of electrons:                             8
  Number of occupied orbitals:                     4
  Number of orbital functions:                    13
  Number of independent orbital functions:        13

 PW_GRID: Information for grid number                                          1
 PW_GRID:                         spherical cutoff                    150.0 a.u.
 PW_GRID: Grid points within cutoff                                       511759
 PW_GRID:   Bounds   1            -62      62                Points:         125
 PW_GRID:   Bounds   2            -62      62                Points:         125
 PW_GRID:   Bounds   3            -62      62                Points:         125
 PW_GRID: Volume element (a.u.^3)  0.5970E-02     Volume (a.u.^3)     11661.1189
 PW_GRID: Grid span                                                    HALFSPACE



 PW_GRID: Information for grid number                                          2
 PW_GRID:                         spherical cutoff                     75.0 a.u.
 PW_GRID: Grid points within cutoff                                       180935
 PW_GRID:   Bounds   1            -45      44                Points:          90
 PW_GRID:   Bounds   2            -45      44                Points:          90
 PW_GRID:   Bounds   3            -45      44                Points:          90
 PW_GRID: Volume element (a.u.^3)  0.1600E-01     Volume (a.u.^3)     11661.1189
 PW_GRID: Grid span                                                    HALFSPACE



 PW_GRID: Information for grid number                                          3
 PW_GRID:                         spherical cutoff                     37.5 a.u.
 PW_GRID: Grid points within cutoff                                        63838
 PW_GRID:   Bounds   1            -32      31                Points:          64
 PW_GRID:   Bounds   2            -32      31                Points:          64
 PW_GRID:   Bounds   3            -32      31                Points:          64
 PW_GRID: Volume element (a.u.^3)  0.4448E-01     Volume (a.u.^3)     11661.1189
 PW_GRID: Grid span                                                    HALFSPACE



 PW_GRID: Information for grid number                                          4
 PW_GRID:                         spherical cutoff                     18.8 a.u.
 PW_GRID: Grid points within cutoff                                        22573
 PW_GRID:   Bounds   1            -22      22                Points:          45
 PW_GRID:   Bounds   2            -22      22                Points:          45
 PW_GRID:   Bounds   3            -22      22                Points:          45
 PW_GRID: Volume element (a.u.^3)  0.1280         Volume (a.u.^3)     11661.1189
 PW_GRID: Grid span                                                    HALFSPACE


RUNNING THE L-BFGS-B CODE

           * * *

Machine precision = 2.220D-16
 N =            36     M =             4
 This problem is unconstrained.

At X0         0 variables are exactly at the bounds

  Step  Update method            Time           Convergence        Total energy
  -----------------------------------------------------------------------------

  Core Hamiltonian energy:                                        10.5602658593
  Hartree energy:                                                 17.0795472553
  Exchange-correlation energy:                                    -3.5547349467

At iterate    0    f= -5.93821D-01    |proj g|=  6.83534D-01

At iterate    1    f= -8.82753D-01    |proj g|=  3.45408D-01

At iterate    2    f= -9.58084D-01    |proj g|=  3.40355D-02

At iterate    3    f= -9.58787D-01    |proj g|=  9.82550D-04

At iterate    4    f= -9.58788D-01    |proj g|=  1.21081D-06

At iterate    5    f= -1.63514D+00    |proj g|=  3.59837D-01

At iterate    6    f= -1.63514D+00    |proj g|=  3.59837D-01

At iterate    7    f= -1.74894D+00    |proj g|=  2.96702D-01

At iterate    8    f= -1.89028D+00    |proj g|=  2.98411D-01

At iterate    9    f= -1.97759D+00    |proj g|=  1.43653D-01

At iterate   10    f= -1.99655D+00    |proj g|=  7.38889D-02

At iterate   11    f= -2.00624D+00    |proj g|=  5.19908D-02

At iterate   12    f= -2.00699D+00    |proj g|=  4.12161D-02

At iterate   13    f= -2.00820D+00    |proj g|=  3.68911D-03

At iterate   14    f= -2.00822D+00    |proj g|=  2.38779D-03

At iterate   15    f= -2.00822D+00    |proj g|=  4.64424D-04

At iterate   16    f= -2.00822D+00    |proj g|=  1.67164D-04

At iterate   17    f= -2.00822D+00    |proj g|=  4.83247D-05

At iterate   18    f= -2.00822D+00    |proj g|=  1.64991D-05

At iterate   19    f= -2.00822D+00    |proj g|=  1.00172D-05

At iterate   20    f= -2.00822D+00    |proj g|=  6.73870D-06

At iterate   21    f= -2.00822D+00    |proj g|=  1.14663D-06

At iterate   22    f= -2.00822D+00    |proj g|=  5.07868D-07
     1  Mixing  0.40            10.25          0.0000137601      -21.0500885159

  Core Hamiltonian energy:                                        10.5602637650
  Hartree energy:                                                 17.0795489652
  Exchange-correlation energy:                                    -3.5547345414

At iterate   23    f= -2.00822D+00    |proj g|=  2.94592D-07
     2  Mixing  0.40             8.76          0.0000083428      -21.0500884950

  Core Hamiltonian energy:                                        10.5602623918
  Hartree energy:                                                 17.0795500688
  Exchange-correlation energy:                                    -3.5547342871

At iterate   24    f= -2.00823D+00    |proj g|=  1.74420D-07
     3  Mixing  0.40             8.59          0.0000041637      -21.0500885103

  Core Hamiltonian energy:                                        10.5602621516
  Hartree energy:                                                 17.0795502580
  Exchange-correlation energy:                                    -3.5547342352

At iterate   25    f= -2.00823D+00    |proj g|=  1.13208D-07
     4  Mixing  0.40             8.73          0.0000025090      -21.0500885094

  Core Hamiltonian energy:                                        10.5602624743
  Hartree energy:                                                 17.0795499948
  Exchange-correlation energy:                                    -3.5547342838

At iterate   26    f= -2.00823D+00    |proj g|=  8.86239D-08
     5  Mixing  0.40             8.82          0.0000015054      -21.0500884985

  Core Hamiltonian energy:                                        10.5602626679
  Hartree energy:                                                 17.0795498235
  Exchange-correlation energy:                                    -3.5547343174
  ys= 0.000E+00  -gs= 0.000E+00 BFGS update SKIPPED

At iterate   27    f= -2.00823D+00    |proj g|=  7.54472D-08
     6  Mixing  0.40             8.71          0.0000009032      -21.0500885099

  *** SCF run converged ***


  Total electronic density (r-space):         -7.9999996787        0.0000003213
  Total core charge density (r-space):         7.9999999989       -0.0000000011
  Total charge density (r-space):                                  0.0000003202
  Total charge density (g-space):                                  0.0000003202

  Overlap energy of the core charge distribution:                  0.0000000000
  Self energy of the core charge distribution:                   -45.1351666838
  Core Hamiltonian energy:                                        10.5602626679
  Hartree energy:                                                 17.0795498235
  Exchange-correlation energy:                                    -3.5547343174
  Total energy:                                                  -21.0500885099


 MO EIGENVALUES AND MO OCCUPATION NUMBERS

                                1            2            3            4
                             0.000000     0.000000     0.000000     0.000000

                             2.000000     2.000000     2.000000     2.000000


                                5            6            7            8
                             0.000000     0.000000     0.000000     0.000000

                             0.000000     0.000000     0.000000     0.000000


                                9           10           11           12
                             0.000000     0.000000     0.000000     0.000000

                             0.000000     0.000000     0.000000     0.000000


                               13
                             0.000000

                             0.000000




 -------------------------------------------------------------------------------
 -                                                                             -
 -                                T I M I N G                                  -
 -                                                                             -
 -------------------------------------------------------------------------------
 SUBROUTINE                        CALLS  CPU TIME T  PERFORMANCE   ELAPSED TIME
 CP2K                                  1     57.10 I       ---             66.00
 Quickstep                             1     57.10 I       ---             66.00
 scf                                   1     57.06 I       ---             65.94
 qs_ks_build_kohn_sham_matrix          6     53.57 I       ---             62.31
 fft_wrap_pw1pw2                      61     32.74 I       ---             37.45
 fft3d_s                              62     23.60 I       ---             26.99
 calculate_rho_elec                    6     22.58 I       ---             25.97
 integrate_v_rspace                    6     14.73 I       ---             17.28
 rs_pw_transfer                       49      5.71 I       ---              6.91
 pw_copy                              54      4.71 I   6.8 Mcopy            5.73
 pw_grid_setup                         4      2.94 I       ---              3.08
 init_grid                             1      2.94 I       ---              3.08
 calculate_xc_potential                6      2.59 I       ---              2.96
 pw_scatter                           30      2.36 I   3.3 Mcopy            3.01
 pw_gather                            31      1.90 I   4.3 Mcopy            1.95
 calculate_rho_core                    1      1.57 I       ---              1.77
 HARTREE                               6      1.48 I       ---              1.87
 pw_grid_sort                          4      0.85 I       ---              0.88
 pw_zero                               6      0.84 I  14.0 Mzero            1.04
 pw_add                               18      0.62 I  14.9 Mflops           0.76
 cp_opt_gerd_f_step,cp_lbfgs_opt      27      0.29 I       ---              0.29
 pw_sumup                              6      0.25 I  24.6 Mflops           0.33
 init_cube_info                        4      0.25 I       ---              0.25
 pw_grid_assign                        4      0.22 I       ---              0.33
 -------------------------------------------------------------------------------


  **** **** ******  **  PROGRAM ENDED AT                Fri Jul  5 16:34:26 2002
 ***** ** ***  *** **   PROGRAM RAN ON                           muralto.cscs.ch
 **    ****   ******    PROGRAM RAN BY                                     fawzi
 ***** **    ** ** **   PROGRAM PROCESS ID                                 10055
  **** **  *******  **  PROGRAM RAN IN                 /home/fawzi/cp2k/tests/QS
